relnotes/integrationtests.migration.md

```diff
@@ -0,0 +1,16 @@
+* Unit and integration tests changes
+
+  * Starting with v2.0.0 Makd will compile all modules from integration tests as part of regular unittest target and stop supplying `-unittest` flag when compiling actual integration tests. This may break compilation of a project so some manual tweaks are required:
+
+    1. If integration test modules don't have module statements or have one with identical module names, it will result in compilation failure.
```

Comment:
"or have one with identical module names" -- does this mean "or multiple modules have the same name" ? (The latter is clearer.)

Reply:
Yes, it seems more clear